webapp/src/containers/modals/LinkModal/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {useEffect} from 'react';
+import {useDispatch} from 'react-redux';
+
```

Comment:
We should remove the empty folders present because of `.gitkeep`. For eg- `containers/action_buttons/.gitkeep`.